vyper/parser/parser.py

```diff
@@ -342,7 +342,7 @@ def parse_func(code, sigs, origcode, global_ctx, _vars=None):
         global_ctx=global_ctx,
         sigs=sigs,
         return_type=sig.output_type,
-        is_constant=sig.const,
+        is_constant='a constant function' if sig.const else None,
```

Comment:
Rather use a boolean value, having a string like that is inconsistent with the other context flags.